### PR TITLE
Fixed Lookup Table Course and Lab

### PR DESCRIPTION
**Pull request by devvspaces**
### Problem

The `Versioned Transactions and Lookup Tables` course has some grammatical errors also in the Lab and I found some deprecated code and incorrect code snippets.

### Summary of Changes

Typos, grammatical errors, code snippets and repos, eliminated async wrappers

### Fixes

- Added keywords for SEO metadata
- Use <Callout> for notes, tips, warnings, and cautions
- Avoided picking favorites by linking to [solana.com/rpc](https://solana.com/rpc) instead of listing specific names
- Added prerequisite installations where necessary
- Fixed Unboxed Lesson and Solution
- Fixed Typo
- Fixed grammatical errors using Grammarly and Hemmingway

### Unboxed PRs

https://github.com/Unboxed-Software/solana-versioned-transactions/pull/2
https://github.com/Unboxed-Software/solana-versioned-transactions/pull/3

### CODE REVIEW

File: content/courses/program-optimization/lookup-tables.md [hunk 1]
```diff
@@ -5,65 +5,81 @@ objectives:
   - Create lookup tables
   - Extend lookup tables
   - Use lookup tables with versioned transactions
-description: "Use large amounts of accounts by using lookup tables."
+description:
+  "Harness a great deal of accounts through the application of lookup tables."
```
Comment (mikemaccana) [inline content/courses/program-optimization/lookup-tables.md:9]: Honestly the old one was simpler. Plain language is important. 

1. Not everyone speaks english. Not everyone knows the word 'harness'. 
2. People want simple clear directions. It should be clear that if they need a lot of accounts in their translations, they should use lookup tables.

> "Use large amounts of accounts in a transaction by using lookup tables."

File: content/courses/program-optimization/lookup-tables.md [hunk 2]
```diff
@@ -5,65 +5,81 @@ objectives:
...
-- **Address Lookup Tables** are accounts used to store addresses of other
-  accounts, which can then be referenced in versioned transactions using a 1
-  byte index instead of the full 32 bytes per address. This enables the creation
-  of more complex transactions than what was possible prior to the introduction
-  of LUTs.
+- **Versioned Transactions** are like different versions of a recipe that allow
+  us to use both old and new ingredients. In this case, the "recipes" are
+  transaction formats. The old version, known as "legacy," and the new versions
+  start from version 0. We made these new versions to work better with a new
+  tool called Address Lookup Tables (LUTs).
```
Comment (mikemaccana) [inline content/courses/program-optimization/lookup-tables.md:24]: > **Versioned Transactions** are like different versions of a recipe that allow
  us to use both old and new ingredients. 
  
 Is too complex. Wedon't need a cooking analogy. People understand versions. The only thing we need to convey is that if they want to use lots of accounts, they need a V0 transaction not a legacy transaction.

File: content/courses/program-optimization/lookup-tables.md [hunk 3]
```diff
@@ -77,18 +93,27 @@ the following parameters:
 You then transform this message object into a version `0` transaction using the
 `compileToV0Message()` method.
 
-```typescript
+Replace the `<YOUR_QUICKNODE_URL_HERE>` placeholder with the URL of your RPC.
+You can get a RPC providers from [Solana RPC](https://solana.com/rpc).
```
Comment (mikemaccana) [inline content/courses/program-optimization/lookup-tables.md:97]: We shouldn't advertise one particular RPC provider. This is mentioned in CONTRIBUTING.md

File: content/courses/program-optimization/lookup-tables.md [hunk 4]
```diff
@@ -77,18 +93,27 @@ the following parameters:
...
+
+```ts
 import * as web3 from "@solana/web3.js";
```
Comment (mikemaccana) [inline content/courses/program-optimization/lookup-tables.md:100]: Just import the bits of web3 you need.

File: content/courses/program-optimization/lookup-tables.md [hunk 5]
```diff
@@ -117,74 +150,81 @@ const transactionSignature = await connection.sendTransaction(transaction);
...
+With versioned transactions, you can use a LUT's address. Then, refer to
+additional accounts with a short 1-byte index instead of their full 32-byte
+addresses. This makes the transaction much smaller and saves a lot of space.
 
 To simplify the process of working with LUTs, the `@solana/web3.js` library
 includes an `AddressLookupTableProgram` class which provides a set of methods to
 create instructions for managing LUTs. These methods include:
 
-- `createLookupTable` - creates a new LUT account
+- `createLookupTable` - creates a new LUT's account
```
Comment (mikemaccana) [inline content/courses/program-optimization/lookup-tables.md:165]: No. The LUT doesn't possess the account, the LUT *is* the account.

**Review by mikemaccana — CHANGES_REQUESTED** [review]
I've left some comments about for the first few parts of this PR, please read them, make changes and then re-submit if you want to get this merged and win this lesson bounty.

**Review by devvspaces — COMMENTED** [review]
I have fixed the issues and made appropriate updates to my PR on Unboxed

**Review by devvspaces — COMMENTED** [review]
Resolved them all thanks @mikemaccana

**Comment by devvspaces** [comment]
hi @mikemaccana i am done with the changes requested here

**Comment by mikemaccana** [comment]
Thanks @devvspaces  ! While you didn't win this particular lesson, there's a lot of good stuff in this PR we'd still love to have your updates. Could you please rebase on top of current main?

**Review by mikemaccana — CHANGES_REQUESTED** [review]
See comment abovere: rebasing!

**Comment by devvspaces** [comment]
@mikemaccana i will do that thanks

**Comment by devvspaces** [comment]
Done that @mikemaccana

**Pull request closed without merging**